### DISCUSSION TITLE
One Retailer on Multiple Seller Apps

### DISCUSSION THREAD

**Discussion by dpandit008** [Open-network-for-digital-commerce/ONDC-Protocol-Specs#27] [Ideas]
If a Retailer is not happy with one Seller App (due to terms and conditions or pricing), and wants to join another Seller; can s/he be hosted in multiple Seller Apps at the same time?

Or it would be like Mobile Number Portability to switch from one Seller App to another Seller App?

**Comment by ukrocks007** [comment]
Just like we can use one bank account with multiple UPI Ids with multiple apps BHIM, PhonePe, Paytm, etc.
In the same way, this interoperability should be there with ONDC.
